README.md

```diff
@@ -2,41 +2,50 @@
-<li><b>Builder Patterns</b> (optional)
+</ul>
+<i>Optional Libraries</i>
+<ul>
+<li><b>Builder Patterns</b>
 <br /><code>sketch://add-library?url=https://www.lightningdesignsystem.com/sketch-library-builder-rss.xml</code></li>
-<li><b>Chart Patterns</b> (optional)
+<li><b>Chart Patterns</b>
 <br /><code>sketch://add-library?url=https://www.lightningdesignsystem.com/sketch-library-chart-rss.xml</code></li>
-<li><b>Rules, Filters, and Logic Patterns</b> (optional)
+<li><b>Rules, Filters, and Logic Patterns</b>
 <br /><code>sketch://add-library?url=https://www.lightningdesignsystem.com/sketch-library-rfl-rss.xml</code></li>
-<li><b>User Engagement Patterns</b> (optional)
+<li><b>User Engagement Patterns</b>
 <br /><code>sketch://add-library?url=https://www.lightningdesignsystem.com/sketch-library-user-engagement-rss.xml</code></li>
-<li><b>Standard Artboard Template</b> (optional)
-<br /><a href="https://github.com/salesforce-ux/design-system-ui-kit/blob/master/Templates/Standard%20Artboards.sketch" title="Standard Artboard Template">Download</a></li>
-</ol>
+</ul>
+<i>Other Resources</i>
+<ul>
+<li><b>Artboard Templates</b>
+<br /><code>sketch://add-library?url=https://www.lightningdesignsystem.com/sketch-library-artboard-template-rss.xml</code></li>
+<li><b>Specification Library</b>
+<br /><code>sketch://add-library?url=https://www.lightningdesignsystem.com/sketch-library-spec-library-rss.xml</code></li>
+<li><b>Key Mobile Product Screens</b>
+<br /><code>sketch://add-library?url=https://www.lightningdesignsystem.com/sketch-library-key-screens-rss.xml</code></li>
+<li><b>Wireframe Library</b>
+<br /><code>sketch://add-library?url=https://www.lightningdesignsystem.com/sketch-library-wireframes-rss.xml</code></li>
+</ul>
 <br />
-<p align="center"><a href="https://github.com/salesforce-ux/design-system-ui-kit/archive/master.zip">or download and install these files manually</a></p>
 <br />
-<br />
-
-----
 
 ## Requirements
 
 ### Sketch
 
 Download the most recent version of [Sketch](https://www.sketchapp.com/).
 <br />
-<i>These files are optimized for Sketch but if you don't have it you can use a free application with less functionality called <a href="https://www.figma.com/" target="_blank" title="Figma">Figma</a> instead</i>
+<i>If you do not have Sketch, you can use the <a href="https://github.com/salesforce-ux/design-system-ui-kit-framerx" target="_blank" title="Framer SLDS UI Kit">Framer SLDS UI Kit</a> or <a href="https://www.figma.com/file/5dgFdCHB6FGjfOPAZEDNVK/Lightning-Design-System-Components-for-Web?node-id=0%3A1" target="_blank" title="Figma SLDS Components for Web">Figma SLDS Components for Web</a> and <a href="https://www.figma.com/file/5dgFdCHB6FGjfOPAZEDNVK/Lightning-Design-System-Components-for-Web?node-id=0%3A1" target="_blank" title="Figma SLDS Icons">Figma SLDS Icons</a> libraries, however, they may not be as up-to-date as the Sketch files.</i>
```

Comment:
`em` gives us the same styles as italics while conveying meaning to folks on screen readers

Reply:
Updated!